Samples/Experimental/ImGui/main.cpp

```diff
@@ -278,28 +273,30 @@ int main()
 
             if (strIndex < arr.size())
             {
-                ImGui::Text(arr.at(strIndex).c_str());
+                ImGui::Text("%s", arr.at(strIndex).c_str());
```

Comment:
Was this a clang-format/clang-tidy change?

Reply:
This is from clang-tidy. It warns about insecure format string stuff without this.